### ISSUE TITLE
Meta: AngularDart Conference MVP

### ISSUE THREAD

**Issue by matanlurey** [dart-lang/webdev#1]
In short: `pub global activate webdev` and then `webdev <command>`.

- [ ] Create a new `angular` project.
- [ ] Serve and debug your project with dartdevc.
- [ ] Generate and run test(s).

Nice to have(s):

- [ ] Replace the hand-written `build.dart`'s for `angular` repo (and friends)

Stretch (blocked on `build_runner`):

- [ ] Build your project with Dart2JS.

**Comment by jakemac53** [comment]
We could support Dart2JS in a similar way to how we support package:test (using create_merged_dir and then running dart2js under that)?

**Comment by matanlurey** [comment]
That would probably work great for M0.

**Comment by jakemac53** [comment]
https://github.com/dart-lang/build/issues/508 is also a blocker for testing I think, otherwise custom html files won't work

**Comment by nshahan** [comment]
We might want to wrap a pub get with the --no-precompile flag so that the pub get runs without having to run the transformer.

**Comment by alorenzen** [comment]
Or we could just remove the transformers?

**Comment by nshahan** [comment]
Yeah... just not sure if you are planning on publishing a version of angular without transformer support in time for the conference.

**Comment by eredo** [comment]
By AngularDart Conference, do you mean there's going to be a separate AngularDart conference or was it the DartConf?

**Comment by nshahan** [comment]
>Yeah... just not sure if you are planning on publishing a version of angular without transformer support in time for the conference.

That comment was in regards to the DartConf that happened in Jan 2018.

**Comment by kevmoo** [comment]
Going to close this out – the feature requests are captured...

**Comment by SYip** [comment]
Are there any feature request for component scaffolding?
Are there any release planner for this project that is publicly avaliable?
